update documentation for -ingress-class (#4542)
nginxinc/kubernetes-ingress: docs/content/configuration/global-configuration/command-line-arguments.md

### -ingress-class `<string>`

> This argument refers to the name of resource `kind: IngressClass`. An IngressClass resource with the name equal to the class must be deployed. Otherwise, the Ingress Controller will fail to start.

Reviewer: ```suggestion
The `-ingress-class` argument refers to the name of the resource `kind: IngressClass`. An IngressClass resource with a name equal to the class must be deployed. Otherwise, NGINX Ingress Controller will fail to start.
```

> The Ingress Controller will processes Ingress resources that belong to its class i.e. have the `ingressClassName` field resource equal to the class and skip the ones without it, but processes all the VirtualServer/VirtualServerRoute/TransportServer resources that do not have the `ingressClassName` field .

Reviewer 2: empty space before "." ?

> The Ingress Controller will process Ingress resources that belong to its class i.e. have the `ingressClassName` field equal to the value of `-ingress-class` and skip the ones without it, but processes all the VirtualServer/VirtualServerRoute/TransportServer resources that do not have the `ingressClassName` field.

Reviewer: ```suggestion
NGINX Ingress Controller will process ingress resources that belong to its class, that is, those that have the `ingressClassName` field equal to the value of `-ingress-class` and skip the ones without it. It will also process all the VirtualServer/VirtualServerRoute/TransportServer resources that do not have the `ingressClassName` field.
```

Avoid Latinisms like i.e., e.g.
